force-app/main/default/lwc/scopedNotification/scopedNotification.js

```diff
@@ -32,6 +35,14 @@ export default class ScopedNotification extends LightningElement {
         return iconName;
     }
 
+    get iconVariant() {
```

Comment:
wouldn't this always return inverse for success, info, and error? is that really what we want?

Reply:
yes those 3 need the icon variant to be inverse anything else does not.